OCI-CLI_reference.md

```diff
@@ -0,0 +1,1218 @@
+
+- **Environment variable:** OCI_CLI_RC_FILE
+
+- **Command-line option:** \--cli-rc-file
+
+**Example:**  
+
+\<*CLI command*\> \--cli-rc-file path/to/my/cli/rc/file
+
+#### Setting up the **oci_cli_rc** file
+
+In a new CLI window, run:  
+
+```sh
+oci setup oci-cli-rc --file path/to/target/file
+```
+
+>**Note:** This command creates the file you specify that includes examples of default command aliases, parameter aliases, and named queries.  
+{:.notice}
+
+#### Working with your configuration file
+
+- [Setting up a default profile](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliconfigure.htm#Specifying_a_Default_Profile)  
+
+  **Section:** OCI_CLI_SETTINGS  
+  **Format:** default_profile=\<*profile name*\>
+
+- [Specifying default values](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliconfigure.htm#Specifying_Default_Values)
+
+  The **\--cli-RC-file** file can be divided into different sections with one or more keys per section.  
+
+  **Keys** are named after command line options, but do not use a leading double hyphen (\--).  
+  For example, the key for **\--image-id** is image-id.  
+
+  You can specify keys for single values, multiple values, and flags.  
+
+- [Specifying command aliases](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliconfigure.htm#Specifying_Command_Aliases)
+
+  There are two types of aliases: *global aliases* and *command sequence aliases*.
+
+- [Specifying option aliases](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliconfigure.htm#Specifying_Option_Aliases)
+
+  **Section:** OCI_CLI_PARAM_ALIASES  
+  **Format:** \<*option alias*\> = \<*original option*\>  
+
+  **Example:**  
+  \--ad = \--availability-domain
+
+  > **Note:** Option aliases are applied globally.  
+  {:.notice}
+
+- [Specifying named queries](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliconfigure.htm#Specifying_Named_Queries)
+
+  If you use the **\--query** parameter to filter or manipulate the output, you can define named queries instead of using a **JMESPath** expression on the command line.  
+
+  **Section:** OCI_CLI_CANNED_QUERIES
+
+- autocomplete
+
+  > **Note:** If you used the **CLI installer**, autocomplete was already installed by default.  
+  >
+  > However, if you manually installed the CLI or only wish to enable autocomplete on a per-session basis, use one of the commands below.  
+  {:.notice}
+
+- **Manual installation -** Run the following command:  
+
+  ```sh
+  oci setup autocomplete
+  ```
+
+- **Session-by-session basis** -- Run the following command:
+
+  ```sh
+  eval "$(_OCI_COMPLETE=source oci)"
+  ```
+
+## Using the CLI
+
+[Reference documentation](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/cliconcepts.htm)
+
+### Basic command line syntax
+
+Most commands must specify a service, followed by a resource type and then an action.  
+
+The basic command line syntax is:  
+
+**oci \<*service*\> \<*type*\> \<*action*\> \<*options*\>**
```

Comment:
Is there a reason all these examples are escaped plain text instead of code spans (single backticks)?

Reply:
In this particular case, I just wanted to be able to style the command syntax to draw attention to those specific elements. For the others, I just wanted to build consistency in presentation throughout the document, since I wasn't sure if the site template styled code spans differently than code blocks.